### PR TITLE
Add batch 8

### PR DESCRIPTION
**Pull request by Yullia**

### CODE REVIEW

File: snapshot.yaml [hunk 1]
```diff
@@ -1168,3 +1168,31 @@
...
+  tag: None
+  url: https://github.com/S1ckB0y1337/Active-Directory-Exploitation-Cheat-Sheet
+- id: aG1sOGFv
+  sha: 9eb5c7b4254ac9abc67e8502c96b7db60d6ddb4e
+  tag: None
+  url: https://gitlab.com/exploit-database/exploitdb.git
+- id: Y0w0J1Fj
+  sha: d294f16426a93c40c79cfb6ca3215648b8549ae4
+  tag: None
+  url: https://github.com/goldfishh/chatgpt-tool-hub
```
Comment (babenek) [inline snapshot.yaml:1198]: MIT License

File: snapshot.yaml [hunk 2]
```diff
@@ -1168,3 +1168,31 @@
...
+  tag: None
+  url: https://github.com/dromara/Sa-Token
+- id: NUNYeQ0n
+  sha: 92ff41acd74158e513ac7b47f05ec2e802730466
+  tag: None
+  url: https://github.com/S1ckB0y1337/Active-Directory-Exploitation-Cheat-Sheet
+- id: aG1sOGFv
+  sha: 9eb5c7b4254ac9abc67e8502c96b7db60d6ddb4e
+  tag: None
+  url: https://gitlab.com/exploit-database/exploitdb.git
```
Comment (babenek) [inline snapshot.yaml:1194]: complicated ..

File: snapshot.yaml [hunk 3]
```diff
@@ -1168,3 +1168,31 @@
...
+  tag: None
+  url: https://github.com/nandorojo/solito
+- id: Ly9pW2hN
+  sha: ecb18f5b45165fc91abda4ae28fa5125f3996d63
+  tag: None
+  url: https://github.com/dromara/Sa-Token
+- id: NUNYeQ0n
+  sha: 92ff41acd74158e513ac7b47f05ec2e802730466
+  tag: None
+  url: https://github.com/S1ckB0y1337/Active-Directory-Exploitation-Cheat-Sheet
```
Comment (babenek) [inline snapshot.yaml:1190]: MIT License

File: snapshot.yaml [hunk 4]
```diff
@@ -1168,3 +1168,31 @@
...
+  tag: None
+  url: https://github.com/Gallopsled/pwntools
+- id: XW1fTVA5
+  sha: b414d36ef1db64d3301dbbe035e2db2f9f55f0a4
+  tag: None
+  url: https://github.com/nandorojo/solito
+- id: Ly9pW2hN
+  sha: ecb18f5b45165fc91abda4ae28fa5125f3996d63
+  tag: None
+  url: https://github.com/dromara/Sa-Token
```
Comment (babenek) [inline snapshot.yaml:1186]: Apache License
                           Version 2.0, January 2004

File: snapshot.yaml [hunk 5]
```diff
@@ -1168,3 +1168,31 @@
...
+  tag: None
+  url: https://github.com/awslabs/git-secrets
+- id: azdhYjhb
+  sha: 27366fdeb8b7770d7331ee51055f8dc5d64ecbd1
+  tag: None
+  url: https://github.com/Gallopsled/pwntools
+- id: XW1fTVA5
+  sha: b414d36ef1db64d3301dbbe035e2db2f9f55f0a4
+  tag: None
+  url: https://github.com/nandorojo/solito
```
Comment (babenek) [inline snapshot.yaml:1182]: MIT License

File: snapshot.yaml [hunk 6]
```diff
@@ -1168,3 +1168,31 @@
...
   tag: 3.7.0-1-gf3693cff
   url: https://github.com/CTFd/CTFd
+- id: eXFgQHAm
+  sha: 5357e18bc27b42a827b6780564ea873a72ca1f01
+  tag: None
+  url: https://github.com/awslabs/git-secrets
+- id: azdhYjhb
+  sha: 27366fdeb8b7770d7331ee51055f8dc5d64ecbd1
+  tag: None
+  url: https://github.com/Gallopsled/pwntools
```
Comment (babenek) [inline snapshot.yaml:1178]: complicated ...

File: snapshot.yaml [hunk 7]
```diff
@@ -1168,3 +1168,31 @@
   sha: f3693cffd03d7006ba8fc6159e7e9bc6dca5f13e
   tag: 3.7.0-1-gf3693cff
   url: https://github.com/CTFd/CTFd
+- id: eXFgQHAm
+  sha: 5357e18bc27b42a827b6780564ea873a72ca1f01
+  tag: None
+  url: https://github.com/awslabs/git-secrets
```
Comment (babenek) [inline snapshot.yaml:1174]: Apache License
                           Version 2.0, January 2004

File: meta/69f42949.csv [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Id,FileID,Domain,RepoName,FilePath,LineStart,LineEnd,GroundTruth,WithWords,ValueStart,ValueEnd,InURL,InRuntimeParameter,CharacterSet,CryptographyKey,PredefinedPattern,VariableNameType,Entropy,Length,Base64Encode,HexEncode,URLEncode,Category
+109495,bab3ed01,GitHub,69f42949,data/69f42949/other/bab3ed01.md,81,81,T,F,9,48,F,F,,,,,4.6825876,,F,F,F,Token
+109496,00eeea58,GitHub,69f42949,data/69f42949/test/00eeea58.lock,11157,11157,F,F,10,138,F,F,,,,,3.9206924,,F,F,F,Facebook Access Token
+109497,2bf8686b,GitHub,69f42949,data/69f42949/test/2bf8686b.cjs,149,149,F,F,32125,32145,F,F,,,,,3.1841838,,F,F,F,AWS Multi
+109498,2bf8686b,GitHub,69f42949,data/69f42949/test/2bf8686b.cjs,149,149,F,F,71516,71536,F,F,,,,,3.2037017,,F,F,F,AWS Multi
```
Comment (babenek) [inline meta/69f42949.csv:5]:
```suggestion
```
hESAAKAKIAUEFSCETA0ACQCAKIAFB//8D - will not be found. even not FP

File: meta/88a0011f.csv [hunk 9]
```diff
@@ -0,0 +1,3 @@
+Id,FileID,Domain,RepoName,FilePath,LineStart,LineEnd,GroundTruth,WithWords,ValueStart,ValueEnd,InURL,InRuntimeParameter,CharacterSet,CryptographyKey,PredefinedPattern,VariableNameType,Entropy,Length,Base64Encode,HexEncode,URLEncode,Category
+109508,b3356305,GitHub,88a0011f,data/88a0011f/other/b3356305.md,1088,1092,F,F,0,0,F,F,,,,,4.192728,,F,F,F,Certificate
```
Comment (babenek) [inline meta/88a0011f.csv:2]:
```suggestion
109508,b3356305,GitHub,88a0011f,data/88a0011f/other/b3356305.md,1088,1092,F,F,,,F,F,,,,,4.192728,,F,F,F,Private Key
```

File: meta/98a9ba4e.csv [hunk 10]
```diff
@@ -0,0 +1,5 @@
+Id,FileID,Domain,RepoName,FilePath,LineStart,LineEnd,GroundTruth,WithWords,ValueStart,ValueEnd,InURL,InRuntimeParameter,CharacterSet,CryptographyKey,PredefinedPattern,VariableNameType,Entropy,Length,Base64Encode,HexEncode,URLEncode,Category
+109907,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,38,38,F,F,17,49,F,F,,,,,3.7181392,,F,F,F,Token
+109908,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,66,66,F,F,26,60,F,F,,,,,4.138502,,F,F,F,Token
+109909,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,130,130,T,F,19,58,F,F,,,,,4.7019434,,F,F,F,Google API Key
+1005285,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,156,156,F,F,25,42,F,F,,,,,0,0,F,F,F,Key
```
Comment (babenek) [inline meta/98a9ba4e.csv]:
```suggestion
109907,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,38,38,T,F,17,49,F,F,,,,,3.7181392,,F,F,F,Token
109908,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,66,66,T,F,26,60,F,F,,,,,4.138502,,F,F,F,Token
109909,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,130,130,T,F,19,58,F,F,,,,,4.7019434,,F,F,F,Google API Key
1005285,99d3c5a8,GitHub,98a9ba4e,data/98a9ba4e/other/99d3c5a8.md,156,156,T,F,25,42,F,F,,,,,0,0,F,F,F,Key
```

**Review by babenek — APPROVED** [review]
Approved

**Review by iuriimet — APPROVED** [review]

**Pull request merged**